src/main/java/org/havenapp/main/SettingsFragment.java

```diff
@@ -115,8 +115,28 @@ public void onCreatePreferencesFix(Bundle bundle, String s) {
             findPreference(PreferenceManager.REGISTER_SIGNAL).setSummary(R.string.register_signal_desc);
         }
 
-        if (preferences.getNotificationTimeMs()>0)
```

Comment:
Overwritten and replaced during the testing phase. Missed when pushing from another machine, replaced in final commit out of place.)